Finish Object Tutorial Chapter 4 (#2125)
MystenLabs/sui: doc/src/build/programming-with-objects/ch4-object-wrapping.md

>:bulb: The fact that you can no longer use a wrapped Sui object means that it's impossible to create circular wrapping behavior, where A wraps B, B wraps C, and C also wraps A.

> At some point, you can then take out the wrapped object and transfer it to your account again. This process is called **unwrapping**. When an object is **unwrapped**, it will become an independent object again, and can be accessed directly on-chain. There is also an important property about wrapping and unwrapping: *The object's ID stays the same after wrapped and then unwrapped*!

Reviewer: ```suggestion
At some point, you can then take out the wrapped object and transfer it to an address. This process is called **unwrapping**. When an object is **unwrapped**, it will become an independent object again, and can be accessed directly on-chain. There is also an important property about wrapping and unwrapping: *The object's ID stays the same across wrapping and unwrapping*!
```
